config/initializers/paperclip.rb

```diff
@@ -0,0 +1,17 @@
+Paperclip::Attachment.default_options.merge!(
+  path: ':rails_root/uploads/:class/:attachment/:id_partition/:style/:filename',
+  url: '/uploads/:class/:id/:style'
```

Comment:
Yeah, I wasn't intending to dig into it now, just letting you know that based on what I know now, it isn't straightforward.